metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -326,16 +326,107 @@ final class Doctor(
           _.element("thead")(
             _.element("tr")(
               _.element("th")(_.text("Build target"))
-                .element("th")(_.text("Scala"))
+                .element("th")(_.text("Type"))
                 .element("th")(_.text("Diagnostics"))
-                .element("th")(_.text("Goto definition"))
-                .element("th")(_.text("Completions"))
-                .element("th")(_.text("Find references"))
+                .element("th")(_.text("Interactive"))
+                .element("th")(_.text("Semanticdb"))
+                .element("th")(_.text("Debugging"))
+                .element("th")(_.text("Java support"))
                 .element("th")(_.text("Recommendation"))
             )
           ).element("tbody")(html => buildTargetRows(html, targetIds))
         )
     }
+
+    // Additional explanations
+    html.element("div")(
+      _.element("p")(
+        _.text(
+          "Diagnostics:"
+        )
+      ).element("ul")(
+        _.element("li")(
+          _.text(
+            s"${Icons.unicode.check} - connected build server showing diagnostics"
+          )
+        )
+          .element("li")(
+            _.text(s"${Icons.unicode.alert} - only syntactic errors")
+          )
+      )
+    )
+
+    html.element("div")(
+      _.element("p")(
+        _.text(
+          "Interactive features (completions, hover):"
+        )
+      ).element("ul")(
+        _.element("li")(
+          _.text(
+            s"${Icons.unicode.check} - supported Scala version"
+          )
+        )
+          .element("li")(
+            _.text(s"${Icons.unicode.error} - unsupported Scala version")
```

Comment:
What do you think of only showing the icons that appear up above? Would that be difficult? It might be nice to for example not show this part at all unless a build target is actually using an unsupported Scala version.

Reply:
Should we also show the explanation for a correct status if everything is correct in the column?

Reply:
I _think_ that still makes sense. Because for example semanticdb may not be super well known, so having a ✅ with information about what it means might help people understand it better.

Reply:
Something along these lines?

![image](https://user-images.githubusercontent.com/3807253/147933232-ae7ce1c0-ede0-4a80-bb49-f5d4cd662f64.png)

Reply:
![image](https://user-images.githubusercontent.com/3807253/147948980-401e3f5a-5130-4f19-b59c-d737d9092208.png)

Ok, this time showing all the existing statuses.

Reply:
Updated the PR with the current version.